Makefile

```diff
@@ -72,7 +74,7 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 test: manifests kustomize generate fmt vet envtest ginkgo  test-docs ## Run tests.
 	@. ./scripts/testing/set-env-vars.sh; \
 	go test -skip=TestAPIs ./... -timeout $(SUITE_TIMEOUT) -coverprofile cover.out -v; \
-	if [ "$(USE_EXISTING_CLUSTER)" == "true" ]; then $(GINKGO) controllers; else $(GINKGO) $(GINKGO_PARALLEL_FLAG) controllers; fi
+	if [ "$(USE_EXISTING_CLUSTER)" == "true" ]; then GINKGO_LABEL_FILTER=$(GINKGO_LABEL_FILTER) $(GINKGO) controllers; else GINKGO_LABEL_FILTER=$(GINKGO_LABEL_FILTER) $(GINKGO) $(GINKGO_PARALLEL_FLAG) controllers; fi
```

Comment:
What do you think about setting the tests as `Pending` to not run them? See https://onsi.github.io/ginkgo/#pending-specs

Reply:
Pushed.